.github/workflows/pr-request-review.yml

```diff
@@ -0,0 +1,19 @@
+name: Request Review
+
+on:
+  issue_comment:
+    types:
+      - created
+
+jobs:
+  request-review:
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    permissions:
+      pull-requests: write
+    if: github.repository_owner == 'cryostatio' && github.event.issue.pull_request && startsWith(github.event.comment.body, '/request_review')
+    steps:
+      - run: |
+          # Adding label 'review-requested' to the PR. Mergify will add reviewers.
```

Comment:
This file doesn't actually add reviewers. Will need to add .mergify.yml file?

Reply:
Might as well add mergify configurations? Need to update the web-client path tho.

https://github.com/cryostatio/cryostat/blob/main/.mergify.yml